content/foundation/governing-board/index.md

```diff
@@ -118,7 +119,7 @@ The current elected representatives are:
 These documents cover the Matrix Governing Board, its purpose, operations, and
 processes. The content is divided into the following sections:
 
-- [**Terms of Reference (TOR)**](https://matrix.org/media/2024-04-governing-board-terms-of-reference.pdf):
+- [**Terms of Reference (TOR)**](https://matrix.org/media/2025-04-governing-board-terms-of-reference.pdf):
```

Comment:
since we're here, let's use an internal link
```suggestion
- [**Terms of Reference (TOR)**](/media/2025-04-governing-board-terms-of-reference.pdf):
```